src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportUnitTest.java

```diff
@@ -41,17 +41,36 @@
                 {"Sample1\tfile1\t"},           // extra tab
                 {"Sample1\nfile"},              // newline instead of tab
                 {"\t"},                         // only tab
-                {"Sample1 file1"},              // non-tab whitespace
-                {"Sample 1\tfile1"},            // white space in a token
+                {"Sample1 file1"},              // 1 column
+                {" name name\tfile1"},          // preceding whitespace
+                {"name name \tfile1"},          // trailing whitespace
         };
     }
 
-    @Test(dataProvider = "getBadTestFiles", expectedExceptions = UserException.BadInput.class)
+    @Test(dataProvider = "getBadSampleNameMapFiles", expectedExceptions = UserException.BadInput.class)
     public void testBadInputFiles(final String text){
         final File sampleFile = IOUtils.writeTempFile(text, "badSampleMapping", ".txt");
         GenomicsDBImport.loadSampleNameMapFile(sampleFile.toPath()  );
     }
 
+    @DataProvider
+    public Object[][] getGoodSampleNameMapFileSyntax(){
+        return new Object[][]{
+                // Note: none of these files are real, these are just valid files syntactically
+                {"Sample1\tsamplePath\n"
+                +"Sample2\tsamplePath", 2},     // normal sample names
+                {"Sample1 001\tFile", 1},          // sample names with whitespace
+                {"name name\tfile1 ", 1},          // trailing whitespace second column
+                {"name name\t file1 ", 1},         // leading and trailing whitespace second column
+        };
+    }
+
+    @Test(dataProvider = "getGoodSampleNameMapFileSyntax")
+    public void testValidSampleFiles(final String text, final int numEntries){
+        final File sampleFile = IOUtils.writeTempFile(text, "badSampleMapping", ".txt");
+        Assert.assertEquals(GenomicsDBImport.loadSampleNameMapFile(sampleFile.toPath()).size(),numEntries);
```

Comment:
Also check the sample names and file names themselves to ensure that they were parsed correctly.